### ISSUE TITLE
Supplemental Heat High Shutoff Temperature column is missing from table DX Heating Coils AHRI 2023

### ISSUE THREAD

**Issue by Nigusse** [NREL/EnergyPlus#10623]
Issue overview
--------------
Table DX Heating Coils AHRI 2023 was created by splitting the data in table DX Heating Coils In PR #10588. Now, table DX Heating Coils has an additional column, `Supplemental Heat High Shutoff Temperature,` which does not exist in the new table `DX Heating Coils AHRI 2023`. These two tables should contain the same set of information. So, the `Supplemental Heat High Shutoff Temperature` column header should be added to the `DX Heating Coils AHRI 2023` table.

### Details
Some additional details for this issue (if relevant):
 - Platform (all, develop)
 - Version of EnergyPlus (if using an intermediate build, include SHA)

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to EnergyPlus Defect Complexity (Github Project)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by Nigusse** [comment]
See below the difference between these two tables:

![image](https://github.com/user-attachments/assets/b0dc5e29-f6eb-47f2-bbb3-dba47a0048d7)

Column `Supplemental Heat High Shutoff Temperature` must be a recent addition. @JasonGlazer, right?

**Comment by JasonGlazer** [comment]
Yes, it was added for the work related to Standard 229 to generate a "RPD" file.

**Comment by Nigusse** [comment]
@JasonGlazer Do you think you'll work on this, or should I include it in one of the PR that I am working on?

**Comment by JasonGlazer** [comment]
The Supplemental Heat High Shutoff Temperature should be done. If you come across a model that does not output that correctly, please let me know.

**Comment by JasonGlazer** [comment]
Or do you mean should it be int he AHRI 2023 table? I don't think it needs to be repeated, do you?

**Comment by Nigusse** [comment]
The last three columns in the current tables are repeated. These two tables should contain the same information, but I don't have a strong opinion either way. Let's leave it as is for now.
